### PR TITLE
Issue 210: change shown tags

### PR DESCRIPTION
**Pull request by texonth**

### CODE REVIEW

File: src/js/blocks/autoFind/autoFindProvider/pageObject.js [hunk 1]
```diff
@@ -11,6 +11,12 @@ const getPackage = (url) => {
       .join(".");
 };
 
+export const getJdiClassName = (label) => {
```
Comment (MariiaNebesnova) [inline src/js/blocks/autoFind/autoFindProvider/pageObject.js:14]: I suggest to move this function to generationClassesMap.js

Comment (texonth) [reply src/js/blocks/autoFind/autoFindProvider/pageObject.js]: Ok, i like to move it

Comment (texonth) [reply src/js/blocks/autoFind/autoFindProvider/pageObject.js]: move it

**Review by Bryant-24 — APPROVED** [review]
PR looks good for me, thanks.

**Review by MariiaNebesnova — APPROVED** [review]
thanks for code!

**Pull request merged**